packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -808,9 +808,12 @@ const processAppExtensions = (
 
             // Copy the Application Extension into the appropriate folder inside application-extensions
             const appExtensionTmpPath = p.join(appExtensionTmp, 'package')
-            const appExtensionDestDir = p.join(appExtensionsDir, appExtensionName)
+            const appExtensionDestDir = p.join(appExtensionsDir, appExtensionName.replace('/', '_'))
```

Comment:
Will this work on windows?

Reply:
Yes, I tested it on Windows, and it works well.